publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RabbitMqProperties.java

```diff
@@ -276,6 +293,9 @@ private void initService() {
         if (channelsCount == null ) {
             channelsCount = Integer.getInteger(getValuesFromSystemProperties(protocol + ".rabbitmq.channelsCount"));
         }
+        if (waitForConfirmsTimeOut == null ) {
+            waitForConfirmsTimeOut = Long.getLong(getValuesFromSystemProperties(protocol + ".rabbitmq.waitForConfirmsTimeOut"));
```

Comment:
Why PropertiesConfig.WAIT_FOR_CONFIRMS_TIME_OUT is not used instead of "protocol + ..."?

Reply:
it was used for publish-cli not for publish-service